components/automate-ui/src/app/modules/desktop/desktop-detail/desktop-detail.component.ts

```diff
@@ -40,11 +37,15 @@ export class DesktopDetailComponent implements OnInit {
     unknown: 'help',
     missing: 'help'
   };
+  private subscription: Subscription;
+  private isDestroyed: Subject<boolean> = new Subject<boolean>();
```

Comment:
Type is inferred and unambiguous, so better to omit:
```suggestion
  private isDestroyed = new Subject<boolean>();
```